### DISCUSSION TITLE
"Compare As" option in Windows XP

### DISCUSSION THREAD

**Discussion by dribgnikcom** [WinMerge/winmerge#1780] [General]
In Windows XP, the "Compare As" functionality seems to be missing in the open dialog.  In Windows 7, there's a small arrow on the compare button, and I am able to select "Binary" for example for a hex editor-type view comparison.  In Windows XP, I just have the button, and the only way to get it to use this view is to use a ".bin" extension for one of the files.

Is this something that's possible to add for XP, or is this a limitation of the API?

Thanks for your great work.

**Comment by sdottaka** [comment]
Unfortunately, this split button has been available since Windows Vista, as shown in the documentation below, so it will fall back to a normal button on Windows XP.

https://learn.microsoft.com/en-us/windows/win32/controls/button-types-and-styles

As shown below, it is possible to display a new Binary Compare window and drag and drop a file into that window, or open it again with the Recompare As menu.

![image](https://user-images.githubusercontent.com/98126/227973029-6d6a7080-2fc0-4b7e-85af-6bc086242d07.png)
![image](https://user-images.githubusercontent.com/98126/227973594-f291a07d-4615-4364-b564-1aaceb5b4e2a.png)
